playbooks/roles/cluster/templates/workflow_survey.json

```diff
@@ -0,0 +1,51 @@
+{
+    "description": "",
+    "name": "",
+    "spec": [
+      {
+        "question_description": "Desired name of Openshift cluster",
+        "min": 0,
+        "default": "",
+        "max": 16,
+        "required": true,
+        "choices": "",
+        "variable": "oo_clusterid",
+        "question_name": "Cluster Name",
+        "type": "text"
+      },
+      {
+        "question_description": "AWS account name",
+        "min": 0,
+        "default": "aws.eng",
+        "max": 100,
+        "required": false,
+        "choices": "{{ integreatly_install_survey_aws_accounts }}",
```

Comment:
As per other comment, might be worth replacing `integreatly` references with `cluster` instead

Reply:
sure, makes sense, done.